neo/Network/P2P/Helper.cs

```diff
@@ -1,10 +1,50 @@
 ﻿using Neo.Network.P2P.Payloads;
+using System;
 using System.IO;
+using System.IO.Compression;
+using K4os.Compression.LZ4;
+using K4os.Compression.LZ4.Streams;
+using Neo.Network.P2P.Payloads;
 
 namespace Neo.Network.P2P
 {
     public static class Helper
     {
+        private static readonly LZ4EncoderSettings CompressSettings = new LZ4EncoderSettings() { CompressionLevel = LZ4Level.L00_FAST };
```

Comment:
The default `LZ4EncoderSettings` (`null`) will use `LZ4Level.L00_FAST`. So we don't need this static field.

Reply:
Here is the default settings: https://github.com/MiloszKrajewski/K4os.Compression.LZ4#stream-compression-settings